nengo/synapses.py

```diff
@@ -450,8 +455,329 @@ class Alpha(LinearFilter):
     tau = NumberParam("tau", low=0)
 
     def __init__(self, tau, **kwargs):
-        super().__init__([1], [tau ** 2, 2 * tau, 1], **kwargs)
         self.tau = tau
+        super().__init__(([1], [tau ** 2, 2 * tau, 1]), analog=True, **kwargs)
+
+    @property
+    def cutoff(self):
+        """Cutoff frequency in Hz; frequencies above this are attenuated."""
+        return np.sqrt(np.sqrt(2) - 1) / (2 * np.pi * self.tau)
```

Comment:
Checked this via https://www.wolframalpha.com/input/?i=%7C1+%2F+%28tau*i*sqrt%28sqrt%282%29+-+1%29+%2F+%28tau%29+%2B+1%29%5E2%7C :+1: